genisys/make_password.py

```diff
@@ -15,6 +15,11 @@ def make_password(self, raw_password, filename):
         secret_password = pbkdf2_sha256.hash(raw_password)
         self.config["users"]["root-login"] = secret_password
 
+        # store updated configuration back into config file
+        parser = YAMLParser
+        config = parser.as_dict(filename)
+        config["users"]["password"] = secret_password
```

Comment:
I believe we are using "Users" (note the capital U) so this may cause an issue as dictionary keys are case-sensitive.